app/views/extensions/index.html.erb

```diff
@@ -172,11 +172,11 @@
         <% else %>
           <% @extensions.each do |ext| %>
             <tr>
-              <td><a class="clickable student-select" data-cud=<%= ext.course_user_datum_id %>><%= ext.course_user_datum.email %></a></td>
-              <td style="padding: 0 5px"><%= ext.infinite? ? "Infinite" : "#{pluralize(ext.days, "day")}" %></td>
-              <td style="padding: 0 5px"><%= ext.infinite? ? "-" : "#{(@assessment.due_at + ext.days.days).strftime('%b %d %Y')}" %></td>
+              <td><a class="clickable student-select" data-cud="{<%= ext.course_user_datum_id %>}"><%= ext.course_user_datum.email %></a></td>
```

Comment:
this change is making the students un-selectable when trying to edit them in the extensions page; when inspecting element, they get rendered like
```
<a class="clickable student-select" data-cud="{8}">user4_AutoPopulated@foo.bar</a>
```
rather than
```
<a class="clickable student-select" data-cud="8">user4_AutoPopulated@foo.bar</a>
```

Reply:
fixed